applications/matter_bridge/README.rst

```diff
@@ -346,15 +341,16 @@ After building the sample and programming it to your development kit, complete t
 
                uart:~$ matter_bridge scan
 
-            The terminal output is similar to the following one:
+            The terminal output is similar to the following one (in case of using a single Bluetooth LE device only one entry will be available):
```

Comment:
```suggestion
            The terminal output is similar to the following one:
```
Since we're already saying it's similar, it is not necessary to say anything about the number. If you feel it is necessary, I'd use the following instead:
```suggestion
            The terminal output is similar to the following one, with an entry for each connected Bluetooth LE device:
```